src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public abstract class Task {
+    protected String description;
+    protected boolean done = false;
```

Comment:
```suggestion
    protected boolean isDone = false;
```

Perhaps change variable name to sound like a boolean?